### DISCUSSION TITLE
Usenet sites in 3rd bar

### DISCUSSION THREAD

**Discussion by dimqua** [Purfview/IMDb-Scout-Mod#62] [Ideas]
Could you please add an option to place Usenet sites in 3rd bar? Or make it the default behavior if you prefer.

**Comment by Purfview** [comment]
I personally don't see point in such feature, if someone wants to PR it then I don't mind.

PS
Maybe better in same bar but split to a new line with sorting func.

**Comment by dimqua** [comment]
Unfortunately I don't have enough skills to implement it as an option. Hopefully no one will complain.

**Comment by Purfview** [comment]
It should be optional.
